src/panels/config/automation/automation-rename-dialog/dialog-automation-rename.ts

```diff
@@ -66,15 +100,23 @@ class DialogAutomationRename extends LitElement implements HassDialog {
         open
         scrimClickAction
         @closed=${this.closeDialog}
-        .heading=${createCloseHeading(
-          this.hass,
-          this.hass.localize(
-            this._params.config.alias
-              ? "ui.panel.config.automation.editor.rename"
-              : "ui.panel.config.automation.editor.save"
-          )
-        )}
+        .heading=${true}
```

Comment:
Make this a string for a11y
```suggestion
        .heading=${this.hass.localize(
              this._params.config.alias
                ? "ui.panel.config.automation.editor.rename"
                : "ui.panel.config.automation.editor.save"
            )}
```